ts/src/pro/kucoinfutures.ts

```diff
@@ -675,6 +780,9 @@ export default class kucoinfutures extends kucoinfuturesRest {
         //     }
         //
         const id = this.safeString (message, 'id');
+        if (!(id in client.subscriptions)) {
```

Comment:
@ttodua why are you doing this? client.subscriptions is indexed by the messagehash not the subscription id

Reply:
if you want to skip it using the id you need to do the indexBy first (done below)

Reply:
ah, accidentally copied from kucoin.ts, they have similar messages, but different handling, yes.